plugins/com.google.cloud.tools.eclipse.appengine.validation/META-INF/MANIFEST.MF

```diff
@@ -11,4 +11,12 @@ Require-Bundle: org.eclipse.core.resources,
   org.eclipse.jdt.core,
  javax.servlet;bundle-version="3.1.0";visibility:=reexport,
  javax.servlet.jsp;bundle-version="2.2.0";visibility:=reexport
-Import-Package: org.eclipse.core.runtime
+Import-Package: com.google.cloud.tools.eclipse.appengine.facets,
+ com.google.common.annotations;version="15.0.0",
+ com.google.common.base;version="15.0.0",
+ com.google.common.collect,
+ com.google.common.io;version="15.0.0",
```

Comment:
Version "15.0.0" doesn't seem right for all of these. Should be "20.0.0" at least. Try removing versions entirely too. (I remember @elharo said something about this before?)